docs/input/docs/usage/cli/installation.md

```diff
@@ -13,8 +13,9 @@ GitVersion can be installed as a [.NET global tool][dotnet-tool] under the name
 [`GitVersion.Tool`][tool] by executing the following in a terminal:
 
 ```shell
-dotnet tool install --global GitVersion.Tool
+dotnet tool install --global GitVersion.Tool --version x.x.x
```

Comment:
Perhaps it's possible to inject the latest version of GitVersion to use here instead of `x.x.x`?

Reply:
`dotnet tool install --global GitVersion.Tool --version 5.*` should suffice, yes?
That would always fetch the latest stable release from major version 5 and up.

Reply:
Yes, good compromise.